fix-readme-links.js

```
@@ -0,0 +1,35 @@
+			console.log(`Processing directory: ${dir}`);
+			for (const file of files) {
+			const filePath = path.join(dir, file);
+			if (fs.lstatSync(filePath).isDirectory()) {
+				replaceLinks(filePath);
+			} else if (file.endsWith(".md")) {
+				let content = fs.readFileSync(filePath, "utf8");
+				console.log(`Processing file: ${filePath}`);
+				// Replace any README.md links with root directory ("/")
+				content = content.replace(/\[([^\]]+)\]\((?:\.\.\/)*README\.md\)/g, (match, linkText) => {
+					console.log(`Replacing README.md link in ${filePath}`);
+					return `[Admin Docs](/)`;  // Preserve original link text if needed
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)